### ISSUE TITLE
Entangled Chalice missing texture

### ISSUE THREAD

**Issue by rafaelkai** [rubensworks/EvilCraft#823]
#### Issue type:

- :bug: Bug 

____

#### Short description:

Entangled Chalice has no texture

#### Versions:

- This mod: 1.0.4
- Minecraft: 1.16.4

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
It has for me. Could you share some more details where you are missing textures?
Can you also share your full forge log?

**Comment by rafaelkai** [comment]
Seems like a mod conflict, here's my log:

https://justpaste.it/9c1o8

<blockquote><img src="/static/other/fav.ico" width="48" align="right"><div><strong><a href="https://justpaste.it/9c1o8">JustPaste.it - Share Text & Images the Easy Way</a></strong></div></blockquote>

**Comment by rafaelkai** [comment]
This line in the log might be related:

[02ene2021 13:54:57.572] [Render thread/INFO] [xaero.map.WorldMap/]: Block file not found: evilcraft:entangled_chalice

**Comment by rubensworks** [comment]
Thanks.
Where does this missing texture occur? When you have this item in your inventory? Or as block in the world?

**Comment by rafaelkai** [comment]
both

**Comment by rubensworks** [comment]
The log line seems unrelated.

I'm putting this on hold, since I'm unable to reproduce this myself.

**Comment by rafaelkai** [comment]
I've reduced the bug to the lastest versions of Xaeros, Botania and Aquatic Upgrade (And their dependecies) using them along Evilcraft. Removing only one of them, doesn't matter which, fixes the bug. Forge 35.1.36. Here are the mods:

https://imgur.com/a/fp0SQHH

<blockquote><img src="https://i.imgur.com/gwzBFHq.jpg?fb" width="48" align="right"><div><img src="//s.imgur.com/images/favicon-16x16.png" height="14"> Imgur</div><div><strong><a href="https://imgur.com/a/fp0SQHH">Imgur</a></strong></div><div>Post with 0 views.</div></blockquote>

**Comment by rubensworks** [comment]
@rafaelkai Thanks, that's definitely helpful. Could you share a zip of these mods?

**Comment by rafaelkai** [comment]
Here you go, thanks for the help

https://drive.google.com/file/d/1eVMuzbcNc8NG9NPNH4q5nDMjueRkxV_m/view?usp=sharing


<blockquote><div><strong><a href="https://drive.google.com/file/d/1eVMuzbcNc8NG9NPNH4q5nDMjueRkxV_m/view?usp=sharing">Meet Google Drive – One place for all your files</a></strong></div><div>Google Drive is a free way to keep your files backed up and easy to reach from any phone, tablet, or computer. Start with 15GB of Google storage – free.</div></blockquote>

**Comment by rubensworks** [comment]
@rafaelkai Can't access the link...

**Comment by rafaelkai** [comment]
You should now I forgot to give you permission, sorry

**Comment by rubensworks** [comment]
Well this is awkward. Using the same exact modpack as you doesn't produce the problem on my end.
So I'm still not able to reproduce the problem unfortunately.
This might be a system-dependent race condition.

Looks like I'll be needing help from someone that is experiencing this bug and can somehow fix it.

**Comment by selened** [comment]
I have experienced this bug too and I also can't seem to reproduce it. There is a chance that upon booting the game and entering a world it'll be missing, but also a chance nothing is wrong at all. One thing I saw is that enabling or disabling a resource pack (any resouce pack), will give the chalice its texture if it was missing. For example, enabling a resource pack that changed the chalice's texture, and then disabling it again seems to give the chalice its texture back
